pinot-common/src/main/java/org/apache/pinot/common/config/RoutingConfig.java

```diff
@@ -25,25 +25,48 @@
 
 
 public class RoutingConfig extends BaseJsonConfig {
+  public static final String PARTITION_AWARE_SEGMENT_PRUNER_TYPE = "partitionAware";
+  public static final String REPLICA_GROUP_INSTANCE_SELECTOR_TYPE = "replicaGroup";
+
+  @Deprecated
```

Comment:
does this mean that all the v2 broker routing approaches compute the final mapping on the fly (no pre-computed routing tables)?

Reply:
Yes.
Removed the deprecated key now, and will deprecate it once the new implementation getting plugged in.